src/WasiConfiguration.cs

```diff
@@ -450,12 +454,12 @@ private static class Native
             public static extern void wasi_config_delete(IntPtr config);
 
             [DllImport(Engine.LibraryName)]
-            public unsafe static extern void wasi_config_set_argv(Handle config, int argc, byte** argv);
+            public unsafe static extern bool wasi_config_set_argv(Handle config, nuint argc, byte** argv);
```

Comment:
I think this is missing the `[return: MarshalAs(UnmanagedType.I1)]` attribute (see #170).
```suggestion
            [return: MarshalAs(UnmanagedType.I1)]
            public unsafe static extern bool wasi_config_set_argv(Handle config, nuint argc, byte** argv);
```